reporting.md

```diff
@@ -499,11 +497,11 @@ Figure: JSON Report Format
     Presently, the only three valid choices are `tlsa`, `sts`, and the literal
     string `no-policy-found`. It is provided as a string.
 * `policy-string`: The string serialization of the policy, whether TLSA record
-    or MTA-STS policy. Any linefeeds from the original policy MUST be replaced with
-    [SP]. TODO: Help with specifics.
-* `domain`: The Policy Domain upon which the policy was applied. For messages
-    sent to `user@example.com` this field would contain `example.com`. It is
-    provided as a string.
+    ([@!RFC6698] section 2.3) or MTA-STS policy. Any linefeeds from the original
+    policy MUST be replaced with space.
+* `domain`: The Policy Domain is the fully qualified DNS domain name ([@!RFC1035])
+    upon which the policy was applied ([@!RFC5890] for international domains). For
```

Comment:
I still don't read this properly. RFC5890 and the "for international domains" reference implies that the "domain" is only the Policy Domain for IDN domains, but of course it's the Policy Domain in all cases. I am not sure an IDN reference is even needed here; I would say something like

> The Policy Domain is the domain against which the MTA-STS or DANE policy is defined

as we say in Terminology. I'm also not sure the 1035 reference really adds much. (If this was a response to comments from the AD review, forgive me; I have not yet read it!)

We could (if we really wanted to) specify whether A form or U form should be used, but I don't see why, since these are mostly for human consumption.